Scripts/import-logs.ps1

```diff
@@ -1,109 +1,259 @@
+        if($database -eq 'Lansa'){
+            $key = $s3BaseKey  + '/MYSQLS/x_err.log'
+        }
+        else{
+                $key = $s3BaseKey + '/' + $database + '/x_err.log'
+        }
+        Write-Host(' Root: ' + $database)
+        Write-Host('Log File: ' + $xErrLogfile)
+        Write-Host('Key: ' + $key)
+        Write-S3Object -BucketName $s3BasePath -Key $key -File $xErrLogfile -Region $region | Out-Default | Write-Host
+    }
+}
+catch{
+    Write-Host "Message: [$($_.Exception.Message)"] --ForegroundColor Yellow
+}
+Write-Host('--------------------------------------------------------------')
 
-#test log files
-$xErrLog = '\tmp\x_err.log'
-foreach ($database in $databases)
-{
-    
-    $databaseRootPath = Join-Path $RootPath $database
-	$xErrLogfile = Join-Path $databaseRootPath $xErrLogPath
-    $key = $totalSummaryPath + '/' + $database + '/x_err.log'
-    Write-S3Object -BucketName $s3BasePath -Key $key -Content $xErrLogfile -Region $region
+
+Write-Host('#Verifier_Test_Report.txt')
+Write-Host('--------------------------------------------------------------')
+try{
+    $verifierTestReport  = 'Verifier_Test_Report.txt'
+    foreach ($database in $databases)
+    {
+        $databaseRootPath = Join-Path $RootPath $database
+        $verifierTestReportFile = Join-Path $databaseRootPath $verifierTestReport
+        $key=''
+        if($database -eq 'Lansa'){
+            $key = $s3BaseKey  + '/MYSQLS/Verifier_Test_Report.txt'
+        }
+        else{
+                $key = $s3BaseKey + '/' + $database + '/Verifier_Test_Report.txt'
+        }
+        Write-Host('Root: ' + $database)
+        Write-Host('Log File: ' + $verifierTestReportFile)
+        Write-Host('Key: ' + $key)
+        Write-S3Object -BucketName $s3BasePath -Key $key -File $verifierTestReportFile -Region $region | Out-Default | Write-Host
+        
+    }
 }
+catch{
+    Write-Host "Message: [$($_.Exception.Message)"] --ForegroundColor Yellow
+}
+Write-Host('--------------------------------------------------------------')
 
-#Verifier_Test_Report.txt
-$verifierTestReport  = 'Verifier_Test_Report.txt'
-foreach ($database in $databases)
-{
-    
-    $databaseRootPath = Join-Path $RootPath $database
-	$verifierTestReportFile = Join-Path $databaseRootPath $verifierTestReport
-    $key = $totalSummaryPath + '/' + $database + '/Verifier_Test_Report.txt'
-    Write-S3Object -BucketName $s3BasePath -Key $key -Content $verifierTestReportFile -Region $region
+
+
+try{   
+    Write-Host('#Verifier_Test_Summary.txt')
+    Write-Host('--------------------------------------------------------------')
+    $verifierTestSummary = 'Verifier_Test_Summary.txt'
+    foreach ($database in $databases)
+    {
+        $databaseRootPath = Join-Path $RootPath $database
+        $verifierTestSummaryFile = Join-Path $databaseRootPath $verifierTestSummary
+        $key=''
+        if($database -eq 'Lansa'){
+            $key = $s3BaseKey  + '/MYSQLS/Verifier_Test_Summary.txt'
```

Comment:
You use this MYSQLS folder in various places. Why? See the DB Types. I think it should be MSSQLS.

Reply:
done